lib/zclient.h

```diff
@@ -73,6 +73,15 @@ typedef uint16_t zebra_size_t;
 #define ZEBRA_FEC_REGISTER_LABEL          0x1
 #define ZEBRA_FEC_REGISTER_LABEL_INDEX    0x2
 
+
+/* Client Graceful Restart */
+#define ZEBRA_CLIENT_GR_CAPABILITIES       0x1
+#define ZEBRA_CLIENT_ROUTE_UPDATE_COMPLETE 0x2
+#define ZEBRA_CLIENT_ROUTE_UPDATE_PENDING  0x4
+#define ZEBRA_CLIENT_GR_DISABLE            0x8
+#define ZEBRA_CLIENT_RIB_STALE_TIME        0x10
+#define ZEBRA_CLIENT_GR_ENABLED(X) (X & ZEBRA_CLIENT_GR_CAPABILITIES)
```

Comment:
if I have GR_DISABLE doesn't ZEBRA_CLIENT_GR_ENABLED return true?  What does that mean?

Reply:
I'm not a fan here we are treating this as both a bitfield and a set of unique capabilities.  Sounds like a bug waiting to happen.  I would recommend just using an enum

Reply:
Sure, I will use enum for this.